packages/next/src/build/webpack/plugins/define-env-plugin.ts

```diff
@@ -63,7 +63,7 @@ function getNextPublicEnvironmentVariables(): DefineEnv {
   for (const key in process.env) {
     if (key.startsWith('NEXT_PUBLIC_')) {
       const value = process.env[key]
-      if (value) {
+      if (value != null) {
         defineEnv[`process.env.${key}`] = value
       }
```

Comment:
```suggestion
      defineEnv[`process.env.${key}`] = process.env[key] ?? ''
```

Since the value is either string or undefined, how about nullish coalescing?